apd/add_json_attribute.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/ruby -w
+
+require 'json'
+
+file_name = ARGV.first.to_s
+json = JSON.parse(File.open(file_name).read)
+
+json.each do |program|
+	program['notes'] = ''
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.